tests/Unit/Domain/Trips/Actions/StoreActionTest.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tests\Unit\Domain\Trips\Actions;
+
+use Tests\TestCase;
+
+class StoreActionTest extends TestCase
```

Comment:
This needs deleting?

Reply:
ughh thank you i forgot to write tests lol. I shall write some this evening.

Reply:
added